### PR TITLE
feat: Add license header scripts to validate and add license to codebase

### PR DESCRIPTION
**Pull request by flacatus**
This PR create a new tool to check license header in codebase and add new license.

Refference issue: https://github.com/eclipse/che/issues/19647

### CODE REVIEW

**Review by AndrienkoAleksandr — APPROVED** [review]

File: scripts/license.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# catch first arguments with $1
+case "$1" in
+ -c|--check-license)
+  echo -e "[INFO] Launching Eclipse Che license header check."
+  validateChectlLicenseHeader
+  ;;
+ -a|--add-license)
+  echo -e "[INFO] Start adding Eclipse Che license headers to code."
+  addLicensetoChectlCode
```
Comment (tolusha) [inline scripts/license.sh:44]: It is basically commands, not flags.
so, pls rework. It should be `license.sh add` or `license.sh check`

File: .github/workflows/automation-check.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+name: Che Machine Exec
+on: [pull_request]
+
+jobs:
+  run:
+    name: Automation checks
```
Comment (tolusha) [inline .github/workflows/automation-check.yaml:17]: license.sh
```suggestion
    name: license
```

File: .github/workflows/automation-check.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#
```
Comment (tolusha) [inline .github/workflows/automation-check.yaml:1]: Pls rename to license.yaml

**Review by tolusha — APPROVED** [review]

**Comment by l0rd** [comment]
@flacatus as a minor comment: this looks more like a chore rather than a feat

**Pull request merged**